File: HelloRun.API/HelloRun.API/Models/Attivita.cs

namespace HelloRun.API.Models
{
    public class Attivita
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public double Km { get; set; }
        public DateTime Data { get; set; } = DateTime.UtcNow;
    }
}

File: hello_run_app/lib/attivita.dart
class Attivita {
  final int id;
  final String nome;
  final double km;
  final DateTime data;

  Attivita({required this.id, required this.nome, required this.km, required this.data});

  factory Attivita.fromJson(Map<String, dynamic> json) {
    return Attivita(
      id: json['id'],
      nome: json['nome'],
      km: (json['km'] as num).toDouble(),
      data: DateTime.parse(json['data']),
    );
  }
}
